lib/mj-svg2png.js

```diff
@@ -0,0 +1,12 @@
+var fs = require('fs');
+var svgpng = require('svg2png');
+
+exports.svg2png = function(svg, result, scale, width, height, callback) {
+  var sourceBuffer = new Buffer(svg, "utf-8");
+  var returnBuffer = svgpng.sync(sourceBuffer, {
```

Comment:
The `sync` version here will cause the mathjax-node process to block during the conversion and not be able to receive additional requests during that time.  We have avoided the use of synchronous calls (except in cases where they should complete immediately, like removing temporary files) so that if mathjax-node is used as part of something like a web service, that service can continue to respond to requests (by queuing mathjax-node typeset calls) even while mathjax-node is handling earlier requests.  This will prevent that from happening during png creation.